generators/app/templates/_gulpfile.js

```diff
@@ -8,9 +8,12 @@ var fs = require('fs');
 // config
 gulp.paths = {
   dist: 'www',
-  jsFiles: ['app/**/*.js', '!app/bower_components/**/*.js'],
+  jsFiles: ['app/**/*.js', '!app/bower_components/**/*.js', '!app/**/*.spec.js'],
```

Comment:
This depends on where you want to keep your test files. The recommended location is in the `test/` directory in the root of your project and that's what globbing patterns are expecting, that's also where the sub-generators put your files. If you want to make the location of the tests adjustable we can discuss it.